core/templates/dev/head/pages/collection-player-page/collection-player-page.directive.ts

```diff
@@ -250,6 +250,24 @@ oppia.directive('collectionPlayerPage', ['UrlInterpolationService',
               ctrl.collectionId);
           };
 
+          ctrl.getExplorationTitleStyle = function(index) {
+            var explorationTitleStyle = {
```

Comment:
This is missing a font attribute, the mock was using Capriola for the titles.

Reply:
Done!